RestSharpClient/ApiClient/ApiService.cs

```diff
@@ -0,0 +1,242 @@
+﻿using Newtonsoft.Json;
+using RestSharp;
+using Newtonsoft.Json.Converters;
+using System.Globalization;
+using System.Reflection.PortableExecutable;
+using System.Net.Mime;
+using RestSharp.Authenticators;
+using RestSharpClient.Model.RestfulBooker;
+using Settings;
+using RestSharpClient.Utils;
+using RestSharp.Serializers;
+using static System.Runtime.InteropServices.JavaScript.JSType;
+using System.Runtime.CompilerServices;
+using Helpers;
+using JsonSerializer = RestSharpClient.Utils.JsonSerializer;
+
+namespace RestSharpClient.ApiClient
+{
+    public partial class ApiService : BaseClient, IApiService, IDisposable
+    {
+        readonly RestClient restClient;
+
+        public RestResponse? Response { get; set; }
```

Comment:
why does this have to be a property and public

Reply:
My bad, was accessing this property earlier, and later added a 'GetResponse' method. Fixed both the issues. Thanks!